packages/devextreme/js/__internal/ui/chat/chat_message_group.ts

```diff
@@ -0,0 +1,137 @@
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+import Avatar from './chat_avatar';
+import MessageBubble from './chat_message_bubble';
+
+const CHAT_MESSAGE_GROUP_CLASS = 'dx-chat-message-group';
+const CHAT_MESSAGE_GROUP_ALIGNMENT_START_CLASS = 'dx-chat-message-group-alignment-start';
+const CHAT_MESSAGE_GROUP_ALIGNMENT_END_CLASS = 'dx-chat-message-group-alignment-end';
+const CHAT_MESSAGE_GROUP_INFORMATION_CLASS = 'dx-chat-message-group-information';
+const CHAT_MESSAGE_TIME_CLASS = 'dx-chat-message-time';
+const CHAT_MESSAGE_NAME_CLASS = 'dx-chat-message-name';
+const CHAT_MESSAGE_BUBBLE_FIRST_CLASS = 'dx-chat-message-bubble-first';
+const CHAT_MESSAGE_BUBBLE_LAST_CLASS = 'dx-chat-message-bubble-last';
+
+export interface MessageGroupOptions extends WidgetOptions<MessageGroup> {
+  messages: Message[];
+  alignment: 'start' | 'end';
+}
+
+class MessageGroup extends Widget<MessageGroupOptions> {
+  _avatar?: Avatar;
+
+  _getDefaultOptions(): MessageGroupOptions {
+    return {
+      ...super._getDefaultOptions(),
+      ...{
+        messages: [],
+        alignment: 'start',
+      },
+    };
+  }
+
+  _getAlignmentClass(): string {
+    const { alignment } = this.option();
+
+    const alignmentClass = alignment === 'start'
+      ? CHAT_MESSAGE_GROUP_ALIGNMENT_START_CLASS
+      : CHAT_MESSAGE_GROUP_ALIGNMENT_END_CLASS;
+
+    return alignmentClass;
+  }
+
+  _initMarkup(): void {
+    const { alignment, messages } = this.option();
+
+    const alignmentClass = this._getAlignmentClass();
+
+    $(this.element())
+      .addClass(CHAT_MESSAGE_GROUP_CLASS)
+      .addClass(alignmentClass);
+
+    super._initMarkup();
+
+    if (alignment === 'start') {
+      const authorName = messages[0].author?.name;
+
+      const $avatar = $('<div>').appendTo(this.element());
+
+      this._avatar = this._createComponent($avatar, Avatar, {
+        name: authorName,
+      });
+    }
+
+    this._renderMessageGroupInformation(messages?.[0]);
+    this._renderMessageBubbles(messages);
+  }
+
+  _renderMessageBubbles(messages): void {
+    messages.forEach((message, index) => {
+      const $bubble = $('<div>');
+
+      const isFirst = index === 0;
+      const isLast = index === messages.length - 1;
+
+      if (isFirst) {
+        $bubble.addClass(CHAT_MESSAGE_BUBBLE_FIRST_CLASS);
+      }
+
+      if (isLast) {
+        $bubble.addClass(CHAT_MESSAGE_BUBBLE_LAST_CLASS);
+      }
+
+      $bubble.appendTo(this.element());
+
+      this._createComponent($bubble, MessageBubble, {
+        text: message.text,
+      });
+    });
+  }
+
+  _renderName(name, $element): void {
+    $('<div>')
+      .addClass(CHAT_MESSAGE_NAME_CLASS)
+      .text(name)
+      .appendTo($element);
+  }
+
+  _renderTime(timestamp, $element): void {
+    const options: Intl.DateTimeFormatOptions = { hour: '2-digit', minute: '2-digit', hour12: false };
+    const dateTime = new Date(Number(timestamp));
```

Comment:
Are we gonna to store the date in 'tick' format?

Reply:
We don't have to think about these little things in the current iteration